packages/kotti-ui/source/kotti-field-toggle/components/ToggleBox.vue

```diff
@@ -30,15 +30,19 @@ export default defineComponent({
 <style lang="scss">
 @import '../../kotti-field/mixins';
 
-@mixin toggle-colors($check-stroke-color, $border-color, $shadow-main) {
+@mixin toggle-colors($border-color, $shadow-main, $background-color) {
```

Comment:
Not sure if we keep arguments in alphabetical order, if so then:
```suggestion
@mixin toggle-colors($background-color, $border-color, $shadow-main) {
```

Reply:
not really, i think those were sorted by importance or so